ambry-tools/src/main/java/com.github.ambry/account/AccountUpdateTool.java

```diff
@@ -117,22 +125,30 @@
   public static void main(String args[]) throws Exception {
     OptionParser parser = new OptionParser();
 
-    ArgumentAcceptingOptionSpec<String> accountJsonFilePathOpt = parser.accepts("accountJsonPath",
+    ArgumentAcceptingOptionSpec<String> accountJsonPathOpt = parser.accepts("accountJsonPath",
         "The path to the account json file. The json file must be in the form of a json array, with each"
             + "entry to be an account in its json form.")
         .withRequiredArg()
-        .describedAs("account_json_file_path")
+        .describedAs("account_json_path")
         .ofType(String.class);
 
+    ArgumentAcceptingOptionSpec<String> accountsToEditOpt = parser.accepts("accountsToEdit",
+        "The names of the accounts to edit, separated by commas. "
+            + "The script will open an editor where you can edit the json for these accounts.")
+        .withRequiredArg()
+        .describedAs("accounts_to_edit")
+        .withValuesSeparatedBy(",");
+
     ArgumentAcceptingOptionSpec<String> zkServerOpt =
         parser.accepts("zkServer", "The address of ZooKeeper server. This option is required.")
             .withRequiredArg()
             .describedAs("zk_server")
             .ofType(String.class);
 
     ArgumentAcceptingOptionSpec<String> storePathOpt = parser.accepts("storePath",
-        "The root path of helix property store in the ZooKeeper. Must start with /, and must not end "
-            + "with /. It is recommended to make root path in the form of /ambry/<clustername>/helixPropertyStore. This option is required.")
+        "The root path of helix property store in the ZooKeeper. "
+            + "Must start with /, and must not end with /. It is recommended to make root path in the form of "
+            + "/ambry/<clustername>/helixPropertyStore. This option is required.")
```

Comment:
this says that this opt is required but I don't think it is passed to `ensureOrExit()`

Reply:
added